products/emberjs.md

```diff
@@ -39,12 +39,20 @@ auto:
 # see https://emberjs.com/releases/lts/
 # - lts(x) = release(x) + 6 weeks
 # - eoas(x) = lts(x) + 36 weeks
-# - eol(x) = lts(x) + 54 months
+# - eol(x) = lts(x) + 54 weeks
 #
 # For non-LTS :
 # - eoas(x) = release(x+1)
 # - eol(x) = release(x+1)
 releases:
+-   releaseCycle: "5.8"
+    releaseDate: 2024-04-20
+    lts: 2024-06-01
+    eoas: 2024-12-28
+    eol: 2025-05-03
```

Comment:
According to https://emberjs.com/releases/lts/ section "LTS schedule":
```suggestion
    lts: 2024-04-15
    eoas: 2024-11-11
    eol: 2025-04-28
```

Reply:
I missed the table, but it appears this release doesn't follow the LTS policy, so I've filed https://github.com/ember-learn/ember-website/issues/1103 to double check.